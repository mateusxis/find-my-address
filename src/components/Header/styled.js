import styled from 'styled-components';

export const SHeader = styled.header`
  margin-top: 30px;
`;

export const STitle = styled.h1`
  margin: 0;
  line-height: 1.15;
  text-align: center;
  font-size: 2rem;
  color: #ffffff;

  @media (min-width: 601px) {
    font-size: 3rem;
  }

  @media (min-width: 961px) {
    font-size: 4rem;
  }
`;
