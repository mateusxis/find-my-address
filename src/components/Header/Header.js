import { SHeader, STitle } from './styled';

const Header = () => (
  <SHeader>
    <STitle>Busque seu endereÃ§o</STitle>
  </SHeader>
);

export default Header;
