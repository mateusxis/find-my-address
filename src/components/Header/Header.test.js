import renderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {
  it('should render Header', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
