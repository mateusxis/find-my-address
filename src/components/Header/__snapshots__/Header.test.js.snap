// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header 1`] = `
<header
  className="sc-bdvvtL hOtWEz"
>
  <h1
    className="sc-gsDKAQ JeTSR"
  >
    Busque seu endere√ßo
  </h1>
</header>
`;
